back/engines/commercial/flag_inappropriate_content/db/migrate/20210521090727_add_inappropriate_content_flag_to_notifications.rb

```diff
@@ -0,0 +1,5 @@
+class AddInappropriateContentFlagToNotifications < ActiveRecord::Migration[6.0]
+  def change
+    add_reference :notifications, :inappropriate_content_flag, foreign_key: {to_table: :flag_inappropriate_content_inappropriate_content_flags}, type: :uuid
```

Comment:
So I guess the notifications changes you suggested are not going through yet?

Reply:
No, for now we decided to keep adding new columns to the `Notifications` table for all new types of relationships they can reference to.